dashboard/app/helpers/aichat_ai_client.rb

```diff
@@ -154,8 +154,7 @@ def get_response_text(stored_messages, new_message, temperature, system_prompt,
 
     messages.each do |message|
       if message['assets'].is_a?(Array)
-
-        messages_with_assets_count +=1 if message['assets'].size > 1
+        messages_with_assets_count +=1 unless message['assets'].empty?
```

Comment:
I think this is a bug where we were only counting multimodal if there was more than one asset (but not if exactly one).

Reply:
Nice catch!